backend/experiment/actions/consent.py

```diff
@@ -18,16 +18,16 @@ def get_render_format(url: str) -> str:
 
 
 def render_html_or_markdown(dry_text: str, render_format: str) -> str:
-    """
-    render html or markdown
+    """render html or markdown
 
-    Parameters:
-        dry_text: contents of a markdown or html file
-        render_format: type of contents, either 'HTML' or 'MARKDOWN'
+    Args:
+        dry_text (str): contents of a markdown or html file
```

Comment:
I thought it wasn't necessary to add types here if it's already in the function description.